src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java

```diff
@@ -319,40 +319,32 @@ public static String getImageUrl(DicomStore store, String studyId) {
     }
 
     private String formatQuery(StudyQuery query) {
-        String allItems = "?includefield=all";
-        if (query == null) {
-            return allItems;
-        }
-
         List<String> parameters = new ArrayList<>();
-        if (isNotBlank(query.getPatientName())) {
-            parameters.add("PatientName=" + urlEncode(query.getPatientName()));
-        }
-
-        if (isNotBlank(query.getPatientId())) {
-            parameters.add("PatientID=" + urlEncode(query.getPatientId()));
-        }
-
-        if (isNotBlank(query.getAccessionNumber())) {
-            parameters.add("AccessionNumber=" + urlEncode(query.getAccessionNumber()));
-        }
-
-        if (query.getStartDate() != null && query.getEndDate() != null) {
-            parameters.add("StudyDate="
-                    + urlEncode(DATE_FORMAT.format(query.getStartDate()))
-                    + "-" + urlEncode(DATE_FORMAT.format(query.getEndDate()))
-            );
-        }
 
-        if (isNotBlank(query.getPhysicianName())) {
-            parameters.add("ReferringPhysicianName=" + urlEncode(query.getPhysicianName()));
-        }
-
-        if (parameters.isEmpty()) {
-            return allItems;
+        if (Objects.nonNull(query)) {
+            if (isNotBlank(query.getPatientName())) {
+                parameters.add("PatientName=" + urlEncode(query.getPatientName()));
+                parameters.add("fuzzymatching=true");
```

Comment:
It's not necessary that whenever the patient name is matched it is a fuzzymatch - we should probably allow the user to choose whether to fuzzymatch or if they want exact matching.

Reply:
is it should be checkbox near patient name field or something else?
![image](https://user-images.githubusercontent.com/878990/58262959-6b9c6c00-7da5-11e9-87f9-0539af6e7131.png)

Reply:
Yup that makes sense.

Reply:
![image](https://user-images.githubusercontent.com/878990/58267568-157ff680-7dae-11e9-9048-df88c94d9548.png)
it looked weird, so i changed it a bit
![image](https://user-images.githubusercontent.com/878990/58267655-42cca480-7dae-11e9-88f9-1dd932bb6ec2.png)

Reply:
Looks good thanks :)